src/Blog/Post/PostService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Blog\Post;
+
+use App\Blog\Entity\Post;
+use App\Blog\Entity\Tag;
+use App\Blog\Tag\TagRepository;
+use App\Entity\User;
+
+final class PostService
+{
+    private PostRepository $repository;
+    private TagRepository $tagRepository;
+
+    public function __construct(PostRepository $repository, TagRepository $tagRepository)
```

Comment:
Maybe it will be more convenient.
```php
public function __construct(PostRepository $repository, TagRepository $tagRepository, UserService $userService)
{
    $this->repository = $repository;
    $this->tagRepository = $tagRepository;
    $this->userService = $userService;
}

public function savePost(Post $model, PostForm $form, User $user = null): void
{
    if ($user === null) {
        if ( $this->userService->getUser() === null)
        {
             throw new \InvalidArgumentException('User not set');
        }
        
        $user = $this->userService->getUser();
    }
    //....
}
```

Reply:
This method should not be called if there is no user.

Reply:
> This method should not be called if there is no user.

This is the way to set the current user as default.

Reply:
> > This method should not be called if there is no user.
> 
> This is the way to set the current user as default.

I don't think it's necessary.